### ISSUE TITLE
link post-upload always redirect to one specific ids box

### ISSUE THREAD

**Issue by Alex-Michie** [izenn/padbox#6]
Really small thing, just that after uploading a json file its linked to your own box rather than the id in the field when i hit upload.

I did get a debug warning message but its likely unrelated. (included for completeness)

```
Processor Output:
Wide character in print at ./pad.pl line 801.
done
```

**Comment by izenn** [comment]
yeah, that is a known issue, but it doesn't seem to affect anything so i've let that particular bug exist, i'll squash it one of these days

**Comment by izenn** [comment]
had a couple minutes so i fixed it in 6d4a50a
